src/Microsoft.ComponentDetection.Detectors/rust/Contracts/CargoMetadata.cs

```diff
@@ -0,0 +1,603 @@
+// <auto-generated />
```

Comment:
Generated by what?

Also nit, but I generally prefer single class per file. But I can live with it if it's autogenerated.

Reply:
Its autogenerated from the output of `cargo metadata` from https://app.quicktype.io/?l=csharp

Reply:
Granted it did not get everything right so I had to make some changes.